src/lib/payload-encoder.ts

```diff
@@ -717,6 +719,24 @@ export class PayloadEncoder {
           return '0x';
         }
       }
+      case 'mstable': {
+        try {
+          const { opType } = data;
+          return web3Coder.encodeParameter(
+            {
+              ParentStruct: {
+                opType: 'bytes',
```

Comment:
Shouldn't this be ABI encoded in uint as expected by the contracts [here](https://github.com/paraswap/paraswap-contracts/pull/21/files#diff-be6f6d91c450fd91649393bea7530463a46ebee8cd67dd170aa614415972c140R18)

Reply:
True, have it localy. That's why it's working for me.